packages/ringcentral-widgets-demo/dev-server/Phone.js

```diff
@@ -250,18 +250,18 @@ export default class BasePhone extends RcModule {
           '/conferenceCall/mergeCtrl',
           '/conferenceCall/dialer/',
           '/calls/active'
-        ].find(path => routerInteraction.currentPath.indexOf(path) !== -1) &&
+        ].find(path => routerInteraction.currentPath === path) &&
```

Comment:
'/conferenceCall/dialer/' still has a query parameter.  Should keep using indexOf ?

Reply:
fixed